<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsImageAction"/>
    <addaction name="separator"/>
    <addaction name="closeAction"/>
    <addaction name="closeAllAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="printPreviewAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="copyAction"/>
    <addaction name="cutAction"/>
    <addaction name="pasteAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Diagram">
    <property name="title">
     <string>&amp;Diagram</string>
    </property>
    <addaction name="simulatorAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="menu_Diagram"/>
   <addaction name="windowMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="itemsDockWidget">
   <property name="windowTitle">
    <string>Shapes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QDiagramShapeToolBox" name="diagramShapeToolBox">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="undoViewDockWidget">
   <property name="windowTitle">
    <string>Undo Stack</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="undoViewDockContentWidget">
    <layout class="QVBoxLayout" name="undoViewDockContentWidgetLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QUndoView" name="undoView" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="propertiesDockWidget">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="propertiesDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QDiagramGraphicsItemPropertiesView" name="propertiesView" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="printAction"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="cutAction"/>
   <addaction name="copyAction"/>
   <addaction name="pasteAction"/>
   <addaction name="separator"/>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="newAction">
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/diagram.new</normaloff>:/icons/diagram.new</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/diagram.save</normaloff>:/icons/diagram.save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/edit.undo</normaloff>:/icons/edit.undo</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/edit.redo</normaloff>:/icons/edit.redo</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/edit.copy</normaloff>:/icons/edit.copy</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/edit.cut</normaloff>:/icons/edit.cut</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/edit.paste</normaloff>:/icons/edit.paste</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="printAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/print</normaloff>:/icons/print</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="printPreviewAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>P&amp;rint Preview</string>
   </property>
  </action>
  <action name="saveAsImageAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as Image</string>
   </property>
  </action>
  <action name="refreshAction">
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/view.refresh</normaloff>:/icons/view.refresh</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="simulatorAction">
   <property name="text">
    <string>Simulator</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/diagr,.open</normaloff>:/icons/diagr,.open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/window.cascase</normaloff>:/icons/window.cascase</iconset>
   </property>
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="tileAction">
   <property name="icon">
    <iconset resource="qdiagramdraw.qrc">
     <normaloff>:/icons/window.tile</normaloff>:/icons/window.tile</iconset>
   </property>
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QUndoView</class>
   <extends>QWidget</extends>
   <header>qundoview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDiagramGraphicsItemPropertiesView</class>
   <extends>QWidget</extends>
   <header>qdiagramgraphicsitempropertiesview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDiagramShapeToolBox</class>
   <extends>QWidget</extends>
   <header>qdiagramshapetoolbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qdiagramdraw.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>saveAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mdiArea</sender>
   <signal>subWindowActivated(QMdiSubWindow*)</signal>
   <receiver>MainWindow</receiver>
   <slot>subWindowActivated(QMdiSubWindow*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>undoAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>undoActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redoAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>redoActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printPreviewAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>printPreviewActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exitActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cutAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cutActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simulatorAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>simulatorActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>copyActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pasteAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>pasteActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openActionTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cascadeAction</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>cascadeSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tileAction</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>tileSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeAction</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeActiveSubWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeAllAction</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeAllSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveActionTriggered()</slot>
  <slot>subWindowActivated(QMdiSubWindow*)</slot>
  <slot>undoActionTriggered()</slot>
  <slot>redoActionTriggered()</slot>
  <slot>printPreviewActionTriggered()</slot>
  <slot>exitActionTriggered()</slot>
  <slot>newActionTriggered()</slot>
  <slot>cutActionTriggered()</slot>
  <slot>simulatorActionTriggered()</slot>
  <slot>copyActionTriggered()</slot>
  <slot>pasteActionTriggered()</slot>
  <slot>openActionTriggered()</slot>
 </slots>
</ui>
